src/main/java/org/folio/linked/data/client/SrsClient.java

```diff
@@ -11,4 +11,7 @@ public interface SrsClient {
 
   @GetMapping(value = "/records/{inventoryId}/formatted?idType=INSTANCE")
   ResponseEntity<Record> getFormattedSourceStorageInstanceRecordById(@PathVariable("inventoryId") String inventoryId);
+
+  @GetMapping(value = "/records/{srsId}/formatted?idType=SRS_RECORD")
```

Comment:
This API is protected by the Permission "source-storage.records.formatted.item.get". See this -> https://github.com/folio-org/mod-source-record-storage/blob/v5.9.0/descriptors/ModuleDescriptor-template.json#L119C14-L119C55. So, let us add "source-storage.records.formatted.item.get" permission to the "modulePermissions" section of our ModuleDescriptor for the `POST /resource` and `PUT /resource/{id}` API.

Also, can you please do one more cleanup? "source-storage.records.formatted.item.get" is the new permission name that replaced the old permission name "source-storage.records.get". See here -> https://github.com/folio-org/mod-source-record-storage/blob/v5.9.0/descriptors/ModuleDescriptor-template.json#L425C20-L425C48. Can you please change all references (there are 3 of them) to "source-storage.records.get" to "source-storage.records.formatted.item.get"  in our module descriptor file?